src/index.ts

```diff
@@ -4,6 +4,13 @@ import * as gitUtils from "./gitUtils";
 import { runPublish, runVersion } from "./run";
 import readChangesetState from "./readChangesetState";
 
+core.warning(
```

Comment:
I've considered using `core.notice` - but I guess this is a "real" deprecation and that deserves a warning